### ISSUE TITLE
[Feature Request] Options to buy a pet through a direct command

### ISSUE THREAD

**Issue by P0lyaris** [MyPetORG/MyPet#1501]
Hi! Is it possible to buy pets not through the gui store, but with the help of a command? But you need a team on behalf of the player, something like /pet shop boys cat.
I know that it is possible to issue a pet on behalf of the admin, but I need a specific command on behalf of the player to be able to check his pet limit. If I give out a pet via /petadmin create, then the limit of pets for a given player will not be checked and he will get an extra pet.

**Comment by Pikachu2p** [comment]
Yes, directly with when entering a command on behalf of the player

**Comment by Pikachu2p** [comment]
Yes, I know about this function, but I would like to make my own icon in the form of a head for each egg icon and arrange it in the deluxe menus plugin. Maybe somehow you can use the heads from the database in the mypet plugin?

**Comment by Jakllp** [comment]
Sadly the eggs are pretty much hardcoded in and as of right now it is not on my priority-list

I might however add that command fairly soon (but not in the next build)

**Comment by Pikachu2p** [comment]
Oh good. Then I will really look forward to this update, as I really need it. Thanks :)
